code/web/src/modules/user/Profile.js

```diff
@@ -24,23 +26,67 @@ const Profile = (props) => (
     </Helmet>
 
     {/* Top title bar */}
-    <Grid style={{ backgroundColor: grey }}>
-      <GridCell style={{ padding: '2em', textAlign: 'center' }}>
+    <Grid style={{ backgroundColor: grey }} justifyRight={true}>
+      <GridCell style={{ padding: '2em', textAlign: 'center', maxWidth: '20vw', marginRight: '24%' }}>
```

Comment:
The Inline styling for this page was really smart. I don't have too much experience with this topic, but what was done to format the different "grids" on the page turned out really nice.